vignettes/subgroup.Rmd

```diff
@@ -51,7 +51,7 @@ raw_data
 #> # ℹ 890 more rows
 ```
 
-Each subgroup level should have adequate representation among all treatment groups at all discrete time points. Otherwise, some marginal means of interest may not be estimable.
+Each categorical subgroup level should have adequate representation among all treatment groups at all discrete time points. Otherwise, some marginal means of interest may not be estimable.
```

Comment:
Happy to migrate to a real dataset when there is a compelling use case. As @yonicd pointed out, current FEV data is simulated. I would also hesitate because we haven't validated `SEX` or `RACE` as valid subgroup variables for FEV in this case. And `brm_simulate_outline()` currently helps because it demonstrates how subgroup simulations can be done.